packages/amplify_authenticator/lib/src/widgets/form_field.dart

```diff
@@ -172,15 +168,15 @@ abstract class AuthenticatorFormFieldState<FieldType, FieldValue,
   @override
   Widget build(BuildContext context) {
     final inputResolver = stringResolver.inputs;
-    final String title = widget.title == null
-        ? inputResolver.resolve(context, widget.titleKey!)
-        : widget.title!;
+    final String? title =
+        widget.title ?? widget.titleKey?.resolve(context, inputResolver);
+
     return Container(
       margin: FormFieldConstants.marginBottom,
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
-          Text(title),
+          if (title is String) Text(title),
```

Comment:
Works but I think I prefer `if (title != null)` and unwrapping it - uglier, but I think it reads more clearly... idk, maybe there's enough context to know that title is a nullable string, and this is a null check. Will let you decide.

Reply:
Yeah, the intention is more clean with `!=null`. Will update.